Update fleets in Getting started guides (#2027)
balena-io/docs: pages/learn/getting-started.md

<img alt="Advanced options" src="/img/common/app/advanced.png" width="80%">

> Once you have finished your image configuration, click the *Download {{ $names.os.lower }}* button. When the download completes, you should have a zipped image file with a name like `{{ $names.company.short }}-First-App-2.47.1+rev1-v10.6.27.img.zip`, where `First-App` is the name you gave your fleet on the dashboard.

Reviewer: ```suggestion
Once you have finished your image configuration, click the *Download {{ $names.os.lower }}* button. When the download completes, you should have a zipped image file with a name like `{{ $names.company.short }}-First-Fleet-2.47.1+rev1-v10.6.27.img.zip`, where `First-Fleet` is the name you gave your fleet on the dashboard.
```

__Note:__ Other authentication methods include using your username and password credentials or obtaining an [authentication token][token] from the dashboard. Authentication tokens come in two types, API tokens, and JSON Web Token (JWT) session tokens. While API tokens do not expire, JWT session tokens do after 7 days.

After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} fleets` command, which should return information about the fleet you created in the previous step. Take a note of the `FLEET NAME` as you'll need this in the next step to push the code to all devices in that fleet.

```shell
$ {{ $names.company.lower }} fleets

> ID    APP NAME   DEVICE TYPE          ONLINE DEVICES DEVICE COUNT
> 98264 First-App  {{ $device.name }}   0              0

Author: ```suggestion
ID    FLEET NAME   DEVICE TYPE          ONLINE DEVICES DEVICE COUNT
98264 First-Fleet  {{ $device.name }}   0              0
```

__Note:__ Other authentication methods include using your username and password credentials or obtaining an [authentication token][token] from the dashboard. Authentication tokens come in two types, API tokens, and JSON Web Token (JWT) session tokens. While API tokens do not expire, JWT session tokens do after 7 days.

> After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} apps` command, which should return information about the fleet you created in the previous step. Take a note of the `APP NAME` as you'll need this in the next step to push the code to all devices in that fleet.

Reviewer: ```suggestion
After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} fleets` command, which should return information about the fleet you created in the previous step. Take a note of the `FLEET NAME` as you'll need this in the next step to push the code to all devices in that fleet.
```

```shell

> $ {{ $names.company.lower }} apps

Reviewer: ```suggestion
$ {{ $names.company.lower }} fleets
```

__Note:__ You may also use git to deploy code to a device. If you wish to deploy via git, see the instructions [here][gitdocs].

> Now to deploy this code to all device(s) in the fleet, use the `{{ $names.company.lower }} push First-App` command replacing `First-App` with the name of your fleet. Ensure you are in the root of the project directory before issuing this command or specify the `--source` option to provide an alternate location of the project directory.

Reviewer: ```suggestion
Now to deploy this code to all device(s) in the fleet, use the `{{ $names.company.lower }} push First-Fleet` command replacing `First-Fleet` with the name of your fleet. Ensure you are in the root of the project directory before issuing this command or specify the `--source` option to provide an alternate location of the project directory.
```

```shell

> $ {{ $names.company.lower }} push First-App

Reviewer: ```suggestion
$ {{ $names.company.lower }} push First-Fleet
```

			             <'
```

> Your application will then be downloaded and executed by all the devices you have connected in your fleet. The first push is slower to deploy, but all subsequent pushes are much faster due to [Docker layer sharing][dockerLayerDocs]. You can see the progress of the device code updates on the device dashboard:

Reviewer: ```suggestion
Your release will then be downloaded and started by all the devices you have connected in your fleet. The first push is slower to deploy, but all subsequent pushes are much faster due to [Docker layer sharing][dockerLayerDocs]. You can see the progress of the device code updates on the device dashboard:
```
